tuna/find_db.py

```diff
@@ -144,9 +144,8 @@ class ConvolutionFindDB(BASE, FindDBMixin):  #pylint: disable=too-many-instance-
                                      name="uq_idx"),)
 
   config = Column(Integer, ForeignKey("conv_config.id"), nullable=False)
-  blobs = relationship("ConvolutionKernelCache",
-                       back_populates="conv_find_db_entries",
-                       cascade="all, delete-orphan")
+
+  kernel_group = Column(Integer, nullable=True)
```

Comment:
please add this column to the mixin if it shows up in both Conv and BN tables. same for the cache tables